### ISSUE TITLE
Hangs with non-beta NV driver v460.79 and Vulkan SDK 1.2.162.0

### ISSUE THREAD

**Issue by BattleAxeVR** [nvpro-samples/vk_raytrace#1]
Hi there, just to let you know, this sample is hanging for me during startup after the debug statement "Create RtCore" in updatePipeline, specifically : vk_raytrace\rtx_pipeline.cpp (line 134).

The other vulkan KHR ray tracing samples seem to work fine with the mainline nvidia VK drivers, so I figured I'd try it here. No dice.

**Comment by missmah** [comment]
> Hi there, just to let you know, this sample is hanging for me during startup after the debug statement "Create RtCore" in updatePipeline, specifically : vk_raytrace\rtx_pipeline.cpp (line 134).
> 
> The other vulkan KHR ray tracing samples seem to work fine with the mainline nvidia VK drivers, so I figured I'd try it here. No dice.

I can confirm this behavior (2x RTX 2080Ti) same drivers and SDK.

**Comment by mklefrancois** [comment]
This is a driver bug, commenting the `ignoreIntersectionEXT` in the AnyHit shader will make the `createRayTracingPipelineKHR()` to pass, but transparency won't work. A new driver with this fix should be available very soon.

**Comment by BattleAxeVR** [comment]
Yay. I'll close this as soon as I get the update. I've had issues with AnyHit stopping working within my own engine when I switched to the mainline drivers so I figured it was part of it. But disabling adding the anyhit shader stage and layout group simply caused a crash later on, in this sample. In my own code, it works but indeed the alpha test stops working. I'll try your workaround in the shader, thanks!
